src/docker-utils.coffee

```diff
@@ -372,5 +372,9 @@ do ->
 			res.status(500).send(err?.message or err or 'Unknown error')
 
 	exports.getImageEnv = (id) ->
-		docker.getImage(id).inspectAsync()
-		.get('Config').get('Env')
+		Promise.try ->
```

Comment:
It didn't, I just felt it looked nicer as "try this block, catch with this block". Changed it back anyways.